lib/coa_env_bootstrapper/runner.rb

```diff
@@ -118,3 +124,12 @@ def generated_pipeline_vars
     end
   end
 end
+
+# rubocop:disable all
+class ::Hash
```

Comment:
this function is not obvious for me, is it  different from deep_merge in `lib/active_support_copy_deep_merge.rb` ? If yes, could you please rename it to avoid confusion ?